lib/daisy_ui_components/tabs.ex

```diff
@@ -0,0 +1,399 @@
+        <button class="tab tab-disabled">Tab 3</button>
+      </div>
+
+  ## Tabs with radio type
+  ## These tabs must provide a title attribute
+      <.tabs>
+        <.tab type="radio" title="Tab 1" name="tabs" />
+        <.tab type="radio" title="Tab 2" name="tabs" active />
+        <.tab type="radio" title="Tab 3" name="tabs" disabled />
+      </.tabs>
+
+  ## Renders:
+      <div role="tablist" class="tabs">
+        <input type="radio" name="tabs" class="tab" aria-label="Tab 1" />
+        <input type="radio" name="tabs" class="tab" aria-label="Tab 2" checked="checked" />
+        <input type="radio" name="tabs" class="tab tab-disabled" aria-label="Tab 3" />
+      </div>
+
+  ## Tabs with label type
+  ## These tabs could have a title attribute or inner block content
+      <.tabs>
+        <.tab type="label" title="Tab 1" name="tabs" />
+        <.tab type="label" title="Tab 2" name="tabs" active />
+        <.tab type="label" title="Tab 3" name="tabs" disabled />
+      </.tabs>
+
+  ## Renders:
+      <div role="tablist" class="tabs">
+        <label class="tab">
+          <input type="radio" name="tabs" />
+          Tab 1
+        </label>
+        <label class="tab">
+          <input type="radio" name="tabs" checked="checked" />
+          Tab 2
+        </label>
+        <label class="tab tab-disabled">
+          <input type="radio" name="tabs" />
+          Tab 3
+        </label>
+      </div>
+
+  ## Tabs with label type and and content
+      <.tabs tabs_style="lift" tabs_position="bottom">
+        <.tab type="label" title="Tab 1" name="my_tabs_with_content" />
+        <.tab_content class="bg-base-100 border-base-300 p-6">Tab content 1</.tab_content>
+        <.tab type="label" title="Tab 2" name="my_tabs_with_content" active />
+        <.tab_content class="bg-base-100 border-base-300 p-6">Tab content 2</.tab_content>
+        <.tab type="label" title="Tab 3" name="my_tabs_with_content" disabled />
+        <.tab_content class="bg-base-100 border-base-300 p-6">Tab content 3</.tab_content>
+      </.tabs>
+
+  ## Renders:
+      <div role="tablist" class="tabs tabs-lift tabs-bottom">
+        <label class="tab">
+          <input type="radio" name="my_tabs_with_content" /> Tab 1
+        </label>
+        <div class="tab-content bg-base-100 border-base-300 p-6">Tab content 1</div>
+
+        <label class="tab tab-active">
+          <input type="radio" name="my_tabs_with_content" checked="checked" /> Tab 2
+        </label>
+        <div class="tab-content bg-base-100 border-base-300 p-6">Tab content 2</div>
+
+        <label class="tab tab-disabled">
+          <input type="radio" name="my_tabs_with_content" /> Tab 3
+        </label>
+        <div class="tab-content bg-base-100 border-base-300 p-6">Tab content 3</div>
+      </div>
+  """
+
+  use DaisyUIComponents, :component
+
+  attr :class, :any, default: nil
+  attr :size, :string, values: sizes()
+  attr :tabs_style, :string, values: ~w(box border lift)
```

Comment:
I would call `border_style` or just `border` on this one

Reply:
Hmm so it could be `<.tab border_style="border">`? I don't like using the word `style` there either. Maybe `display_style`? We could also make each style a boolean value. I think that's how those classes work. So it could be `border=true` or `lift="true"` or something like that

Reply:
that's a good point. I know that in daisyUI they introduced these border styles in a lot of components in the new version so would be good to have some similar attributes across components. 

I usually use the Boolean assigns when we can stack these classes, I think stacking these border elements is not possible but I really like how the API would like from your example. I would say let's do the Boolean attributes

Reply:
@phcurado I updated `tabs_position` to `position` and converted `tabs_style` to boolean. I noticed that setting `<tabs lift>` would work, but `<.tabs lift="true">` didn't. I updated `maybe_add_class` utility function to accept string "true" in addition to boolean since I think HTML can rendered with string "true" in some cases. The tests seem to all pass, but let me know if there are issues with that.

Reply:
all good